core/Ajax.php

```diff
@@ -453,26 +450,42 @@ public static function view_client_list($clientId = '')
                 $html .= 'Create new list';
                 $html .= '</option>';
                 $html .= '<option class="ajax-call" disabled >';
-                $html .= '---';

```

Comment:
This is to change the way the client's list are being displayed. We split it into Single opt-in and Confirmed opt-in, where the confirmed opt-in is disabled and in the bottom.